packages/e2e-tests/fixtures/blocks/core__navigation-menu-item.html

```diff
@@ -1,3 +1,3 @@
 <!-- wp:navigation-menu-item {"label":"WordPress","destination":"https://wordpress.org/"} -->
-<a class="wp-block-navigation-menu-item" href="https://wordpress.org/">WordPress</a>
```

Comment:
Should we keep the `wp-block-navigation-menu-item` class? It's typical that every block has a class like this. They're often useful for third parties.

Reply:
TBH I don't know why it went away but the fixtures were generated without it.